src/modules/vendor/components/programs/cards/templates/MultipleChoiceQuestion.vue

```diff
@@ -73,8 +73,11 @@ export default {
       );
     },
     requiredAnswerIsMissing (index) {
+      if (index === 1 && !this.card.qcmAnswers[0].label) return true;
+      if (index === 1 && this.card.qcmAnswers[2].label) return false;
+
       return index < 2 && this.card.qcmAnswers.filter(a => a.label).length < this.answersLengthInDb &&
-        this.$v.card.qcmAnswers.$error && !this.card.qcmAnswers[index].label;
```

Comment:
Il n'y a pas besoin du `this.$v.card.qcmAnswers.$error` ?

Reply:
C'est en partie de sa faute s'il y avait un comportement étrange. ça vient peut-être du fait que je comprends pas bien comment il fonctionne